### ISSUE TITLE
Cannot build without Git repository

### ISSUE THREAD

**Issue by a-tinsmith** [SIPp/sipp#270]
Steps to reproduce:
1. Download SIPp source code.
2. Try to build.

Error produced:
`make[1]: *** No rule to make target `include/version.h', needed by `src/call.cpp'.  Stop.`

**Comment by wdoekes** [comment]
Steps to find similar bug reports:

https://github.com/SIPp/sipp/issues?utf8=%E2%9C%93&q=is%3Aissue%20version.h

Steps to fix:

Download the *right* file. Did you see the NOTE on the releases page here https://github.com/SIPp/sipp/releases ?

> NOTE Please download sipp-3.5.1.tar.gz from the link below instead of Source code (tar.gz). They are both source code, but the latter one is a snapshot of the git v3.5.1 tag that won't build because it requires the git metadata as well.

**Comment by wdoekes** [comment]
Hm.. I should probably add that comment to the Makefile instead..

**Comment by a-tinsmith** [comment]
Thank you for reply.
Sorry for the confusion. Yes, I saw the release notes. However, I downloaded `3.6-dev` version as I needed `play_dtmf` feature, which is unavailable in v3.5.x.

How about a stub `version.h` file that exists in the code tree and overwritten by the `build.sh`? This way people who download latest sources still can build without repo and those who build from Git-controlled location will have the file updated by git anyway.
